[DOCS] Adds Deploying the model section to regression and classification conceptual (#1273)
elastic/stack-docs: docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

the impact of the different loss function parameters.

[[dfa-regression-deploy]]
=== Deploying the model

The model that you created is stored as {es} documents in internal indices. In 
other words, the characteristics of your trained model are saved and ready to be 
deployed and used as functions. The <<ml-inference,{infer}>> feature enables you 
to use your model in a preprocessor of an ingest pipeline or in a pipeline 

> aggregation of a search query to get a prediction on your data.

Reviewer: It's not imperative, but "get a prediction on your data" sounds like a one-time thing, which I don't think is the case for pipelines at least. I suggest something like this:
```suggestion
aggregation of a search query to make predictions about your data.
```
